### PR TITLE
feat(mdc-toggle-button): New toggle button component in material-experimental

### PR DESCRIPTION
**Pull request by gschrag**
Add new component as wrapper to new mdc segmented-button component.

### CODE REVIEW

File: src/material-experimental/mdc-toggle-button/toggle-button.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+  @Output() readonly change: EventEmitter<SegmentDetail> = new EventEmitter<SegmentDetail>();
+
+  @ContentChildren(MatToggleButtonSegment, {
+    descendants: true
+  }) _segments: QueryList<MatToggleButtonSegment>;
+
+  constructor(
+    readonly _elementRef: ElementRef,
+    public _changeDetectorRef: ChangeDetectorRef
```
Comment (kseamon) [inline src/material-experimental/mdc-toggle-button/toggle-button.ts:76]: readonly

File: src/material-experimental/mdc-toggle-button/toggle-button-segment.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    getAttr: (_attrName) => null,
+    setAttr: (_attrName, _value) => undefined,
+    addClass: (_className) => undefined,
+    removeClass: (_className) => undefined,
+    hasClass: (_className) => false,
+    notifySelectedChange: (_selected) => undefined
+  };
+
+  constructor(
+    public _changeDetectorRef: ChangeDetectorRef,
```
Comment (kseamon) [inline src/material-experimental/mdc-toggle-button/toggle-button-segment.ts:72]: readonly

Comment (kseamon) [reply src/material-experimental/mdc-toggle-button/toggle-button-segment.ts:72]: Also, a leading _ implies private, so make it private if possible

**Comment by andrewseguin** [comment]
Closing this draft since we do not currently have MDC support for the button toggle. Hopefully soon we will be able to implement this

**Pull request closed without merging**